.github/workflows/bundle-size.yml

```diff
@@ -8,10 +8,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2-beta
-      with:
-        fetch-depth: 1
+    - uses: actions/checkout@v2
     - uses: preactjs/compressed-size-action@v1
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
-        pattern: "{build/**/*.js,build/**/*.css}"
```

Comment:
The `pattern` input is required because [ours doesn't match the default pattern](https://github.com/preactjs/compressed-size-action/blob/d85e1f35d37b4df8a5f29c2f06209d70a0578702/action.js#L54).